pkgs/build-support/docker/default.nix

```diff
@@ -434,6 +434,8 @@ rec {
         if [[ -n "$fromImage" ]]; then
           echo "Unpacking base image..."
           tar -C image -xpf "$fromImage"
+          # Do not import the base image configuration
+          rm -f image/*.json
```

Comment:
I think it should be enough to delete the `manifest.json`. Which is new since [docker/moby image spec v1.2](https://github.com/moby/moby/blob/master/image/spec/v1.2.md).
The problem is that the `manifest.json` describes which layers should be used and `buildImage` doesn’t create such a file.
So it definitely makes sense to not import the `manifest.json` from the base image 👍. I don’t think it is necessary to delete all json files.